src/amcl/src/amcl_node.cpp

```diff
@@ -1282,24 +1145,24 @@ AmclNode::laserReceived(const sensor_msgs::LaserScanConstPtr& laser_scan)
       pf_matrix_t pose_cov;
       if (!pf_get_cluster_stats(pf_, hyp_count, &weight, &pose_mean, &pose_cov))
       {
-        ROS_ERROR("Couldn't get stats on cluster %d", hyp_count);
+        RCLCPP_ERROR(get_logger(), "Couldn't get stats on cluster %d", hyp_count);
```

Comment:
There should be a name associated with the logger, IMHO. I'd go with something like `get_logger("amcl")` This applies everywhere.